src/hooks/use-toast/useToast.tsx

```diff
@@ -0,0 +1,9 @@
+import { useContext } from 'react';
+
+import { ToastContext } from '@/hooks/use-toast/toast-context';
+
+const useToast = () => {
```

Comment:
```suggestion
const useToast = (): ToastContext => useContext(toastContext)
```
and move it to file with context

Reply:
Isn't relevant with new updates